styleguide.md

```diff
@@ -0,0 +1,123 @@
+Hello! Thank you for choosing to help contribute to the SendGrid documentation site! There are many ways you can contribute and help is always welcome.  When writing documentation, we ask you to consider the following style guidelines. 
+
+* [Style](#style)
+* [Terminology](#terminology)
+* [Punctuation](#punctuation)
+* [Capitalization](#capitalization)
+* [Warning and info blocks](#warning-and-info-blocks)
+* [Screenshots](#screenshots)
+
+
+<a name="style"></a>
+## Style
+
+We strive to make our documentation concise, clear, organized, and scannable. 
+
+* Use conversational language and industry-standard terms when possible.
+* Contractions are OK and preferred.
+* Use bulleted lists and numbered steps where applicable. 
+
+When giving a user an action item, put the UI element involved in the action in **bold**. Avoid using the word "button" in the step and simply refer to the name on the UI element. 
```

Comment:
This seems awkward, maybe: `When documenting a UI action, make the action **bold**`